[PyTorch] [Memory profiler] Early return if qualified name is invalid

chowarfb opened the pull request:
Summary: Return early if we can easily determine the operator qualified name is invalid before attempting to retrieve the schema. In particular "::" should always be present. Quick estimate shows that this is >50x faster (100 us -> 2 us).

Test Plan: CI

Differential Revision: D47562587

aaronenyeshi reviewed (approved):
LGTM, thank you for catching this bug!

chowarfb committed:
[PyTorch] [Memory profiler] Early return if qualified name is invalid (#105495)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/105495

Return early if we can easily determine the operator qualified name is invalid before attempting to retrieve the schema. In particular "::" should always be present. Quick estimate shows that this is >50x faster (100 us -> 2 us).

Test Plan: CI

Reviewed By: aaronenyeshi, xintwfb

Differential Revision: D47562587

fbshipit-source-id: ac0f67a4ab89621b5fdc00be2896482d447f40b0
